Mobile/ShopLens/Droid/Activities/ShoppingCartActivity.cs

```diff
@@ -19,62 +19,66 @@
 
 namespace ShopLens.Droid
 {
+    // TODO: Make voice interaction in shopping cart fluent.
```

Comment:
I haven't gotten round to implementing voice interaction in shopping cart because Tomas has to use Utterance Listener and I don't want to steal his task.

Reply:
Without Utterance Listener I won't be able to implement voice interactions...